fxdc/config.py

```
@@ -126,14 +249,73 @@ def __init__(self, data):
 
             Config.add_class("MyClass", class_=MyClass)"""
 
+        def generate_meta_data(class_: type) -> dict[str, dict[str, Any]]:
```

Ruff (0.12.2)

`generate_meta_data` is too complex (15 > 10)

(C901)